person_class.rb

```diff
@@ -1,22 +1,15 @@
-# Represents a person with attributes like name and age.
-class Person
-  attr_reader :id, :age
+class Person < Nameable
   attr_accessor :name
+  attr_reader :age
```

Comment:
- [ ] Since age is a getter and setter variable you should add it to your attr_accessor :+1: